identity/auth-starter/src/main/java/io/camunda/identity/authentication/CurrentUserService.java

```diff
@@ -5,11 +5,10 @@
  * Licensed under the Camunda License 1.0. You may not use this file
  * except in compliance with the Camunda License 1.0.
  */
-package io.camunda.identity.usermanagement;
+package io.camunda.identity.authentication;
```

Comment:
Do we need `identity` in the folder structure / class names? This way we have `identity` and `authentication` as (sub)domains. If we consider `identity` as the top level domain here, then it would make sense to split the internals in `authentication` and `authorization`. Otherwise we could just have a top level domain `authentication` and omit `identity`. What do you think?

Reply:
Initially had included `identity` because its our realm and Identity is still a component in the stack in the same way we have `io.camunda.operate...` etc so I'd prefer to keep it, but I can see how we could remove this and have a flat level of `io.camunda.authentication`. Ultimately I don't feel strongly either way so I will change it if you have stronger feelings than me :D

Reply:
> and Identity is still a component
That is also a reason to not use `identity` here, so it is not confused with the application/component in the end. On a scale of 1-10 I am at about 6.5 that we should not use identity here, but just keep it simple as `io.camunda.authentication`

Reply:
Ok, I've changed it to `io.camunda.authentication`